packages/mongo-nav/src/project-nav/ProjectNav.spec.tsx

```diff
@@ -169,4 +171,51 @@ describe('packages/mongo-nav/src/project-nav', () => {
 
     testForAlerts(alerts, true);
   });
+
+  describe('when the date is before MongoDB World', () => {
+    test('Stitch is displayed in the ProjectNav', () => {
+      const productName = displayProductName();
```

Comment:
You should mock this, cuz after May 4th this test will fail.

Reply:
Smart 👍